kivy/core/window/__init__.py

```diff
@@ -1193,6 +1193,13 @@ def on_motion(self, etype, me):
                 self.dispatch('on_touch_up', me)
                 FocusBehavior._handle_post_on_touch_up(me)
 
+            elif not etype:
```

Comment:
Very nice catch with the multitouch, you are a great tester. I didn't expect such an issue at all. I put back the button requirement in sdl2 event and moved my cursor movement event to a new method. Patch will come soon when i get to a computer again.